spec/scheduling-tasks.md

```diff
@@ -272,49 +362,105 @@ Issue: [=Run steps after a timeout=] doesn't necessarily account for suspension;
 [whatwg/html#5925](https://github.com/whatwg/html/issues/5925).
 
 <div algorithm>
-  To <dfn>select the scheduler task queue</dfn> for a {{Scheduler}} |scheduler| given an
-  {{AbortSignal}} or null |signal|, and a {{TaskPriority}} or null |priority|:
-
-  1. If |priority| is null, |signal| is not null and [=implements=] the {{TaskSignal}} interface,
-     and |signal| [=TaskSignal/has fixed priority=], then set |priority| to |signal|'s
-     [=TaskSignal/priority=].
-  1. If |priority| is null and |signal| is not null and [=implements=] the {{TaskSignal}} interface,
-     then
+  To <dfn>schedule a yield continuation</dfn> for {{Scheduler}} |scheduler| given
+  {{SchedulerYieldOptions}} |options|:
+
+  1. Let |result| be [=a new promise=].
+  1. Let |abortOption| be |options|["{{SchedulerYieldOptions/signal}}"] if it [=map/exists=],
+     otherwise null.
+  1. Let |priorityOption| be |options|["{{SchedulerPostTaskOptions/priority}}"] if it
+     [=map/exists=], otherwise null.
+  1. If |abortOption| is null and |priorityOption| is null, then set |abortOption| to
+     "{{SchedulerSignalInherit/inherit}}" and set |priorityOption| to
+     "{{SchedulerSignalInherit/inherit}}".
+  1. Otherwise if |abortOption| is "{{SchedulerSignalInherit/inherit}}" and |priorityOption| is null,
+     then set |priorityOption| to "{{SchedulerSignalInherit/inherit}}".
+  1. Let |state| be the result of [=computing the scheduling state from options=] given |scheduler|,
```

Comment:
"compute the scheduling state from options"

Reply:
Same question as above.